GWB_Green_Domain_Call.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Create a Green Web Foundation Green Domain GET to get the data from the Database
+ * Details: https://datasets.thegreenwebfoundation.org/daily_snapshot/greendomain
+ */
+function newGWFRequest(url="web.dev.com"){
+    Logger.log(url)
+    let regex = new RegExp(/^(?:https?:\/\/)?(?:[^@\n]+@)?(?:www\.)?([^:\/\n?]+)/img);
+    let origin_regex = "";
+    try{
+      origin_regex = regex.exec(url)[1]
+    }
+    catch(e){
+      throw "Error in newGWFRequest"
+    }
+    Logger.log(origin_regex)
```

Comment:
The log message isn't very clear. Can we add more info, or if this was just for debugging, remove the call?

Reply:
done!